src/screens/AddWalletSheet.tsx

```diff
@@ -182,7 +182,7 @@ export const AddWalletSheet = () => {
   };
 
   const onPressWatch = () => {
-    analytics.track('Tapped "Add an existing wallet"');
+    analytics.track('Tapped "Watch an Ethereum Address"');
```

Comment:
do we need to merge the new & old events together somehow & is that even possible?

Reply:
Don't believe this. I think this event was supposed to be what I changed it to. It's what one of the tickets was.